Added tests for Team model methods

makarman opened the pull request:
Added test for to_dict method
Added test for create method
Added test for update method
Added test for get_by_id method
Added test for delete_by_id method

## Code reviewers

- [ ] @dmytrorevak
- [ ] @IvanAndrukh
- [ ] @JjuliaP
- [ ] @makarman
- [ ] @noobhunters
- [ ] @Silent777
- [ ] @Staryj812

### Second Level Review

- [ ] @ssOleg
- [ ] @lhalam


## CHECK LIST
- [ ]  СI passed
- [ ]  Сode coverage >=95%
- [ ]  PR is reviewed manually again (to make sure you have 100% ready code)
- [ ]  All reviewers agreed to merge the PR
- [ ]  I've checked new feature as logged in and logged out user if needed
- [ ]  PR meets all conventions

ssOleg commented on eventually/tests/unittests/team/testmodel.py:
```diff
@@ -0,0 +1,174 @@
+        test_team_dict1 = {
+            'id': 101,
+            'name': 'bananaboys',
+            'description': 'krakaziabra',
```
Please change all strange strings like `bananaboys` and `krakaziabra` to `some name`, `some description` ...

makarman replied:
done

ssOleg commented on eventually/tests/unittests/team/testmodel.py:
```diff
@@ -0,0 +1,174 @@
+        test_team_dict2 = {
+            'id': 102,
+            'name': 'bananadogs',
+            'description': 'krakaziabra mama',
```
for example, `bananadogs`, `krakaziabra mama` to `some name1`, `some description1` ..etc.

makarman replied:
done

makarman committed:
Added tests for Team model methods

Added test for to_dict method
Added test for create method
Added test for update method
Added test for get_by_id method
Added test for delete_by_id method
